pkg/controllers/awsloadbalancercontroller/serviceaccount.go

```diff
@@ -63,6 +64,7 @@ func desiredAWSLoadBalancerServiceAccount(namespace string, controller *albo.AWS
 			Namespace: namespace,
 			Name:      fmt.Sprintf("%s-%s", controllerResourcePrefix, controller.Name),
 		},
+		AutomountServiceAccountToken: aws.Bool(true),
```

Comment:
Now that we are using a non default value we have to update the functions which verifies if the service account needs an update.

Reply:
Yeah, my bad, I forgot this. Added.